packages/core/core-flows/src/draft-order/workflows/confirm-draft-order-edit.ts

```diff
@@ -114,19 +114,25 @@ export const confirmDraftOrderEditWorkflow = createWorkflow(
       throw_if_key_not_found: true,
     }).config({ name: "order-items-query" })
 
-    const lineItemIds = transform(
+    const { removedLineItemIds } = transform(
```

Comment:
comment: We should be able to write a test for this no?

Reply:
@oli - added